src/main/java/core/basesyntax/GeometricFigureSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class GeometricFigureSupplier {
+    public GeometricFigure getRandomFigure() {
```

Comment:
Consider defining the `Random` object as a class-level variable rather than creating a new instance in every method call for better performance.